bindings/node/etc/build-static.sh

```diff
@@ -34,6 +33,4 @@ run_cmake --build . --target install --config RelWithDebInfo
 popd #./
 
 # build the `mongodb-client-encryption` addon
-# note the --unsafe-perm parameter to make the build work
```

Comment:
Why did you remove `--unsafe-perm`?  Is it no longer relevant?

Reply:
Yep, the PR linked is closed, we also don't run as a root user in CI